jme3-core/src/main/java/com/jme3/math/Matrix3f.java

```diff
@@ -1210,7 +1210,7 @@ public int hashCode() {
      */
     @Override
     public boolean equals(Object o) {
-        if (!(o instanceof Matrix3f) || o == null) {
+        if (!(o instanceof Matrix3f)) {
```

Comment:
Note: while this is ok for this class because it's final, the normal idiom would be:
if( o == null || o.getClass() != getClass() ) return false;
....because .equals() should always be an exact class match.

Reply:
Normalizing the idiom would be a separate PR, correct?

Reply:
Correct.

Reply:
I opened issue #1289.